docs/workflows/react-native-specific-workflow-steps/react-native-unit-test.md

```diff
@@ -0,0 +1,67 @@
+---
+title: React Native Unit Test
+description: Learn how to run unit tests with Jest for your React Native projects easily with Appcircle, ensuring high-quality code and improved app performance.
+tags: [react native, mobile, workflow, step, unit, test, jest, unit test]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# React Native Unit Test
+
+This component runs all the unit tests in your project written with [Jest](https://jestjs.io/docs/tutorial-react-native) integration. When this step is completed, it generates a test report file in `JUnit.xml` format. You can view these test results in detail using Appcircle's **Test Report** component. To generate detailed Test Reports. Please visit our [Test Reports Component documentation](/workflows/react-native-specific-workflow-steps/test-reports-react-native).
+
+For detailed information for continuous testing. Please visit our [React Native Continuous Testing documentation](/continuous-testing/react-native-testing/react-native-unit-test-with-jest).
+
+### Prerequisites
+
+The workflow steps that need to be executed before running the **React Native Unit Test** workflow step, along with their respective reasons, are listed in the table below.
+
+| Prerequisite Workflow Step                                                                 | Description                                                                                                                                                                            |
+|--------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [**Git Clone**](/workflows/common-workflow-steps#git-clone)                                | Clone the selected repository to the build machine.                                                                                                                                    |
+| [**Install Node**](/workflows/react-native-specific-workflow-steps#install-node)           | This step will install Node modules for your application.                                                                                                                              |
+| [**NPM/Yarn Commands**](/workflows/react-native-specific-workflow-steps/npm-yarn-commands) | This step install the [NPM](https://www.npmjs.com/) or [Yarn](https://www.npmjs.com/package/yarn) package manager to install specific dependencies for your React Native applications. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4443-rnUnitFlowOrder.png'/>
```

Comment:
We should update the screenshot since the component description got updates while PR was in review.

<img width="1018" alt="Screenshot 2024-11-02 at 08 26 10" src="https://github.com/user-attachments/assets/bcb9f7c9-6ac7-4853-a4cd-949749032c41">

Reply:
I updated all related screenshots for both UI and Unit test comps @csonuryilmaz